modules/cnv-subscribing-to-hco-catalog.adoc

```diff
@@ -12,28 +12,36 @@ namespace access to the {CNVProductName} Operators.
 
 .Prerequisites
 
-* `OperatorGroup` and `CatalogSource` Custom Resource Definition objects (CRDs),
-both created in the `openshift-cnv` namespace
+* Create a project called `openshift-cnv`.
 
 .Procedure
 
 . Open a browser window and navigate to the {product-title} web console.
 
 . Navigate in the web console to the *Operators → OperatorHub* page.
```

Comment:
I would drop "in the web console" and replace to:  Navigate to the *Operators → OperatorHub* page. or In the left menu navigate to the ...

Reply:
Good catch, that was leftover from before the first step existed.